### PR TITLE
Report slightly more when no platform is detected.

### PR DESCRIPTION
**Pull request by isoos**
- #824
- added `package:webdriver` to the end2end tests to track the improvements on this

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,6 +1,7 @@
 ## 0.19.1
 
 * Upgraded dependencies: `analyzer` and `json_serializable`.
+* Report slightly more when no platform is detected.
```
Comment (sigurdm) [inline CHANGELOG.md:4]: You could give a hint of what is being reported.

File: test/goldens/end2end/webdriver-3.0.0.json_report.md [hunk 2]
```diff
@@ -0,0 +1,82 @@
...
+
+See [package layout](https://dart.dev/tools/pub/package-layout#examples) guidelines on how to add an example.
+</details>
+
+## 0/20 Support multiple platforms
+
+### [x] 0/20 points: Supports 0 of 2 possible platforms (native, js)
+
+
+Consider supporting these prerelease platforms:
```
Comment (sigurdm) [inline test/goldens/end2end/webdriver-3.0.0.json_report.md:29]: This line reads weirdly...

File: test/goldens/end2end/webdriver-3.0.0.json_report.md [hunk 3]
```diff
@@ -0,0 +1,82 @@
...
+
+<details>
+<summary>
+Package not compatible with SDK dart
```
Comment (sigurdm) [inline test/goldens/end2end/webdriver-3.0.0.json_report.md:33]: We should end with a full stop `.`.

File: test/goldens/end2end/webdriver-3.0.0.json_report.md [hunk 4]
```diff
@@ -0,0 +1,82 @@
...
+</summary>
+
+Because it is not compatible with any of the supported runtimes: native-aot, native, js
```
Comment (sigurdm) [inline test/goldens/end2end/webdriver-3.0.0.json_report.md:36]: We should end with a full stop `.`.

Comment (sigurdm) [reply test/goldens/end2end/webdriver-3.0.0.json_report.md:36]: We really should document somewhere what this means, and link.
As the package author there is not much to go on.

**Review by sigurdm — APPROVED** [review]
This is an improvement, but still the explanation feels lacking.

**Comment by isoos** [comment]
Added notes to the tracking issue, to be addressed later.

**Pull request merged**